src/app/core/config.py

```diff
@@ -0,0 +1,13 @@
+import os
+from pydantic import AnyHttpUrl, BaseSettings
+
+class Settings(BaseSettings):
```

Comment:
We should have two newlines around this top level class definition according to [PEP8](https://peps.python.org/pep-0008/#blank-lines).